### ISSUE TITLE
drake-related functions

### ISSUE THREAD

**Issue by shirdekel** [MilesMcBain/spacemacs_cfg#11]
Thanks for sharing this, it's been very useful for trying out Spacemacs/ESS.

I can't seem to find your settings for `{drake}`-related functions such as `r_make()` and source all packages (like you have in your VS Code config). Any guidance on how to set this up?

**Comment by MilesMcBain** [comment]
They've been mostly a work thing so on the windows branch?

**Comment by shirdekel** [comment]
Thanks! I see them now.

Did you symlink your private folder? I want to keep mine under version control, and this is what internet seems to suggest to do.

**Comment by shirdekel** [comment]
Also, it seems as though [these lines](https://github.com/MilesMcBain/spacemacs_cfg/blob/8afa422c0dfb562bed1f692c8773844227ee431a/private/ess/packages.el#L176-L178) are the only Windows-specific ones. Is that true? I essentially want to use that file, but with a Mac.

**Comment by MilesMcBain** [comment]
Yes I think so for that file. Also this for the .spacemacs: https://github.com/MilesMcBain/spacemacs_cfg/blob/windows/spacemacs.el#L350

The best branches to fork off are either `windows` or `3.00`. `windows` has had a bit more love and care recently.

I don't symlink it but that is a really good idea. I just started doing this with my VSCode config.

**Comment by shirdekel** [comment]
Great, thanks!

Another thing: [a while ago you mentioned](https://twitter.com/MilesMcBain/status/1294126421044891649) that you have a way of dealing with loading all functions in a project using ESS. I may have misunderstood this, but does this imply that there is an '`ess-eval-all-functions-in-project`'-type of function? Or do you simply not use this kind of command?
